pages/what-we-do/council-selected-restoration-component.md

```diff
@@ -10,5 +10,10 @@ eleventyNavigation:
 ## Council-Selected Restoration Component
 
 Congress directed the Council to administer two different sources of RESTORE Act funding. The first is called the Council-Selected Restoration Component. These funds are used for ecosystem restoration across the Gulf. It is referred to as the Council-Selected Restoration Component because funding is approved by a vote among the Council members. The Council periodically approves Council-Selected Restoration Component funding for batches of projects and programs in what is called a Funded Priorities List (FPL). The Council occasionally amends these FPLs to add additional funding or otherwise modify previously-approved activities. These approvals are done by a Council vote, after the Council seeks public input. FPLs are developed through collaboration among the state and federal Council members.
-
+<br>
+<img src="/img/Buckets Graphic.png" alt="Clean Water Act Penatlies Graph" loading="lazy">
```

Comment:
This works, but let's try to use Markdown syntax where possible so we aren't mixing in HTML unless it's critical.